harness/determined/_experiment_config.py

```diff
@@ -27,6 +27,7 @@ def mixed_precision_enabled(self) -> bool:
     def averaging_training_metrics_enabled(self) -> bool:
         return bool(self["optimizations"]["average_training_metrics"])
 
+    # TODO (DET-3798): Remove input_from_dataflow in 0.12.14.
```

Comment:
No need to put a hard version on this.  We can just have the comment say "remove this after known customers have migrated" or just "remove this at some point" and put details in the jira ticket.

Reply:
done